mdtraj/core/residue_names.py

```diff
@@ -20,35 +20,849 @@
+                     'TBG': 'V',
+                     'TBM': 'T',
+                     'TCQ': 'Y',
+                     'TCR': 'W',
+                     'TDD': 'L',
+                     'TFQ': 'F',
+                     'TH6': 'T',
+                     'THC': 'T',
+                     'THO': 'X',
+                     'THR': 'T',
+                     'THZ': 'R',
+                     'TIH': 'A',
+                     'TMB': 'T',
+                     'TMD': 'T',
+                     'TNB': 'C',
+                     'TNR': 'S',
+                     'TOQ': 'W',
+                     'TPH': 'X',
+                     'TPL': 'W',
+                     'TPO': 'T',
+                     'TPQ': 'Y',
+                     'TQI': 'W',
+                     'TQQ': 'W',
+                     'TRF': 'W',
+                     'TRG': 'K',
+                     'TRN': 'W',
+                     'TRO': 'W',
+                     'TRP': 'W',
+                     'TRQ': 'W',
+                     'TRW': 'W',
+                     'TRX': 'W',
+                     'TRY': 'W',
+                     'TST': 'X',
+                     'TTQ': 'W',
+                     'TTS': 'Y',
+                     'TXY': 'Y',
+                     'TY1': 'Y',
+                     'TY2': 'Y',
+                     'TY3': 'Y',
+                     'TY5': 'Y',
+                     'TYB': 'Y',
+                     'TYI': 'Y',
+                     'TYJ': 'Y',
+                     'TYN': 'Y',
+                     'TYO': 'Y',
+                     'TYQ': 'Y',
+                     'TYR': 'Y',
+                     'TYS': 'Y',
+                     'TYT': 'Y',
+                     'TYW': 'Y',
+                     'TYX': 'X',
+                     'TYY': 'Y',
+                     'TZB': 'X',
+                     'TZO': 'X',
+                     'UMA': 'A',
+                     'UN1': 'X',
+                     'UN2': 'X',
+                     'UNK': 'X',
+                     'VAD': 'V',
+                     'VAF': 'V',
+                     'VAL': 'V',
+                     'VB1': 'K',
+                     'VDL': 'X',
+                     'VLL': 'X',
+                     'VLM': 'X',
+                     'VMS': 'X',
+                     'VOL': 'X',
+                     'WLU': 'L',
+                     'WPA': 'F',
+                     'WRP': 'W',
+                     'WVL': 'V',
+                     'X2W': 'E',
+                     'XCN': 'C',
+                     'XCP': 'X',
+                     'XDT': 'T',
+                     'XPL': 'O',
+                     'XPR': 'P',
+                     'XSN': 'N',
+                     'XX1': 'K',
+                     'YCM': 'C',
+                     'YOF': 'Y',
+                     'YTH': 'T',
+                     'Z01': 'A',
+                     'ZAL': 'A',
+                     'ZCL': 'F',
+                     'ZFB': 'X',
+                     'ZU0': 'T',
+                     'ZZJ': 'A'}
```

Comment:
Are all of these codes derived amino acids?  I don't think they are.  Single-letter codes should be separated from amino acid identification, IMO